.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,43 @@
+name: 📚 Documentation
+description: Report an issue related to documentation
+title: "docs: "
+labels: [documentation, 'status: needs triage']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for your help to improve kernl.
+        
+        This template is strictly used for documentation requests. 
+
+        For example:
+        - Adding or modifying documentation.
+        - Adding or modifying tutorials.
+        - ...
+
+        > This issue should not touch the code base.
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: A clear and concise description of what the issue is.
+    validations:
+      required: true
+
+  - type: checkboxes
+    attributes:
+      label: Self-service
+      description: |
+        If you feel like you could contribute to this issue, please check the box below. This lets us know that someone wants to work on it.
+        If you check this box, please send a Pull Request within 7 days of a maintainer's approval so we can delegate this task.
+      options:
+        - label: I would be willing to contribute to this issue myself.
+
+  - type: checkboxes
```

Comment:
utile une checkbox pour ça ?

Reply:
This is a practice seen to identify a person ready to really contribute. Tell me if it's overkill or too restrictive.